src/handle_provide_parameter.c

```diff
@@ -6,12 +6,75 @@ static void handle_address_to(ethPluginProvideParameter_t *msg, artblock_paramet
     printf_hex_array("Address To: ", ADDRESS_LENGTH, context->address_to);
 }
 
+static void handle_merkle_proof(ethPluginProvideParameter_t *msg, artblock_parameters_t *context) {
+    memset(context->merkle_proof, 0, sizeof(context->address_to));
+    memcpy(context->merkle_proof, &msg->parameter, PARAMETER_LENGTH);
+    printf_hex_array("Merkle Proof: ", PARAMETER_LENGTH, context->merkle_proof);
```

Comment:
It's worth noting that PARAMETER_LENGTH is 32, which means we will at most display 32 bytes of the Merkle proof since that's all will fit on the Ledger

Reply:
good callout!
- This seems totally fine, as there isn't any security risk with a user not understanding the details of their Merkle proof
- Might be worth adding a dev comment here explaining your callout